beacon-chain/rpc/eth/light-client/handlers.go

```diff
@@ -378,7 +355,7 @@ func (s *Server) getLightClientEventBlock(ctx context.Context, minSignaturesRequ
 
 	// Loop through the blocks until we find a block that satisfies minSignaturesRequired requirement
 	var numOfSyncCommitteeSignatures uint64
-	if syncAggregate, err := block.Block().Body().SyncAggregate(); err == nil && syncAggregate != nil {
+	if syncAggregate, err := block.Block().Body().SyncAggregate(); err == nil {
```

Comment:
We can assume it's not nil when there is no error. Checking everything for nil everywhere is over-defensive in my opinion

Reply:
> We can assume it's not nil when there is no error. Checking everything for nil everywhere is over-defensive in my opinion

then we should do it here too:
https://github.com/prysmaticlabs/prysm/blob/e588deddb9680240a1eaef1e32354c3613c13c22/beacon-chain/rpc/eth/light-client/handlers.go#L374